### DISCUSSION TITLE
Cant use non-camelCase field handles as default handles.

### DISCUSSION THREAD

**Discussion by J-Protz** [solspace/craft-freeform#891] [Feature Requests]
**Describe the bug or issue you're experiencing**
I am rebuilding a site for a client, in their craft 3 version there are handles such as "C_BusPhone" , however in Craft 4 while rebuilding I noticed that it camelcases all handles in the field creator/editor. In the form itself, I can change it to not be camelcase however this feels unintuitive and clunky.
**Steps to reproduce**

1. Set up a Craft 4.5(maybe earlier) version with Freeform 4.1.9 (probably earlier too)
2. Make a new field in the "fields" section with underscores and capital letters, etc.

**Expected behavior**
The field will allow me to save an unconventional handle. Maybe even an config setting to allow me to do so.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.5
- Freeform Version: 4.1.9
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh Install

**Additional context**
The field handles are used for a third party that the data gets forwarded to, otherwise I would be fine with this behavior, however if I forget to update the handle on every new form, it may not send the data over correctly

**Comment by kjmartens** [comment]
Thanks for your feedback @J-Protz. This is a great idea. I will run this by our team and we will likely add support for a Handle generator behavior setting to control this in a future version of Freeform. 🙂

**Reply by J-Protz** [reply]
Thanks!

**Comment by kjmartens** [comment]
This will be available with the next version of Freeform (4.1.14+). 🙂
